main.go

```diff
@@ -81,21 +81,36 @@ func main() {
 					Usage:  "URL which receives a POST on state change",
 					EnvVar: "SHAWARMA_URL",
 				},
+				cli.BoolFlag{
+					Name:   "disable-notifier, d",
+					Usage:  "Enable/Disable state change notification",
+					EnvVar: "SHAWARMA_DISABLE_STATE_NOTIFIER",
+				},
+				cli.IntFlag{
+					Name:   "listen-port, l",
+					Value:  8099,
+					Usage:  "Default port to be used to start the http server",
+					EnvVar: "LISTEN_PORT",
```

Comment:
I think this env variable name should be `SHAWARMA_LISTEN_PORT` to agree with the other environment variables.

Reply:
thanks for the review and testing @brantburnett , just updated this env var name to `SHAWARMA_LISTEN_PORT` and updated the README file accordingly